src/components/colorscale/attributes.js

```diff
@@ -9,6 +9,8 @@
 'use strict';
 
 var colorbarAttrs = require('../colorbar/attributes');
+var counterRegex = require('../../lib/regex').counter;
```

Comment:
`regexCounter` may be a better naming here.

Reply:
That's how we export it here:

https://github.com/plotly/plotly.js/blob/c820abb37a5b68387f5d31696db592586f7cebc1/src/lib/index.js#L132-L133

so, I'll pass.